.requirements

```diff
@@ -24,9 +24,9 @@ ATC_ROUTER=ffd11db657115769bf94f0c4f915f98300bc26b6 # 1.6.2
 SNAPPY=23b3286820105438c5dbb9bc22f1bb85c5812c8a # 1.2.0
 
 KONG_MANAGER=nightly
-NGX_WASM_MODULE=96b4e27e10c63b07ed40ea88a91c22f23981db35
+NGX_WASM_MODULE=9b1b2c760f73827fc08ade3a936a89fa5473f8fa
```

Comment:
Just an FYI I juuuust merged https://github.com/Kong/kong/pull/13765, so you can drop the deps changes and rebase to pick them up from master.